### ISSUE TITLE
How to quickly train the model based on the original model for the audio data of the new speaker？

### ISSUE THREAD

**Issue by joewale** [joaoantoniocn/AM-MobileNet1D#2]
How to quickly train the model based on the original model for the audio data of the new speaker？

**Comment by joaoantoniocn** [comment]
I would advise to retrain the entire dataset again

**Comment by joewale** [comment]
the efficiency of training would be lower, and can't do online training

**Comment by joaoantoniocn** [comment]
You can also try to do transfer learning with the model, there are some tutorial of how to do it for computer vision models, you will have to adapt it to your situation.
In the case of AM-MobileNed1D you would have to retrain only the model classifier, but I haven't tried it yet.

model.classifier = nn.Sequential(
            nn.Dropout(0.2),
            nn.Linear(self.last_channel, num_classes[0]),
            nn.LogSoftmax(dim=1),
        )

So, you will have to change the 'num_classes[0]' to your new number of classes.

**Comment by joewale** [comment]
got it，I should have a try. thanks
